portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,26 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Thu!</h1>");
+    //response.setContentType("text/html;");
+    //response.getWriter().println("<h1>Hello Thu!</h1>");
+    response.setContentType("application/json;");
+
+    ArrayList<String> listValues = new ArrayList<String>(); 
```

Comment:
listValues doesn't seem to accurately describe what the ArrayList. comments would be a better name. Also, when initializing the ArrayList, 'new ArrayList<>()' will do the trick

Reply:
![image](https://user-images.githubusercontent.com/44590322/84604791-23436680-ae5e-11ea-9221-539098f0b454.png)
@TsueiCat Got it